### DISCUSSION TITLE
How to increase API call more then 500 requests

### DISCUSSION THREAD

**Discussion by karkyra** [apache/airflow#23067] [Q&A] [answered]
### Apache Airflow version

2.1.1

### What happened

When I am API calling for users with limit 1000, it returns only 500  requests. How or where can I increase limit more then 500 requests? Looks like by default airflow API returns max 500 requests.

I am running following command but always return 500 requests: `https://airflow_url/api/v1/users?limit=1000` 

### Operating System

browser or any linux/mac cli

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by mik-laj** [comment, marked as answer]
> [](https://airflow.apache.org/docs/apache-airflow/stable/security/api.html#page-size-limit)Page size limit
> To protect against requests that may lead to application instability, the stable API has a limit of items in response. The default is 100 items, but you can change it using `maximum_page_limit` option in `[api]` section in the `airflow.cfg` file.

https://airflow.apache.org/docs/apache-airflow/stable/security/api.html
